src/components/TodoItem/TodoItem.tsx

```diff
@@ -0,0 +1,55 @@
+import cn from 'classnames';
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  todos: Todo[]
+  todo: Todo
+  setTodos: (toggle: Todo[]) => void
+  deleted: () => void
+  loadingTodosIds: number[]
+};
+
+export const TodoItem: React.FC<Props> = ({
+  todo,
+  deleted,
+  todos,
+  setTodos,
+  loadingTodosIds,
+}) => {
+  const loaded = loadingTodosIds.includes(todo.id);
+  const toggleTodo = (id: number) => {
+    const toggle = todos.map(todoItem => (todoItem.id === id
+      ? { ...todoItem, completed: !todoItem.completed } : todoItem));
+
+    setTodos(toggle);
+  };
+
+  return (
+    <div className={cn('todo', { completed: todo.completed })}>
+      <label className="todo__status-label">
+        <input
+          type="checkbox"
+          className="todo__status"
+          checked={todo.completed}
+          onClick={() => toggleTodo}
```

Comment:
not fixed 
![image](https://github.com/mate-academy/react_todo-app-add-and-delete/assets/70009205/0b31d47e-8f20-4289-aec7-6998c7bedce7)